packages/flutter/lib/src/widgets/animated_switcher.dart

```diff
@@ -84,30 +85,33 @@ typedef Widget AnimatedSwitcherLayoutBuilder(List<Widget> children);
 ///   @override
 ///   Widget build(BuildContext context) {
 ///     return new Material(
-///       child: Column(
-///         mainAxisAlignment: MainAxisAlignment.center,
-///         children: <Widget>[
-///           new AnimatedSwitcher(
-///             duration: const Duration(milliseconds: 200),
-///             transitionBuilder: (Widget child, Animation<double> animation) {
-///               return new ScaleTransition(child: child, scale: animation);
-///             },
-///             child: new Text(
-///               '$_count',
-///               // Must have this key to build a unique widget when _count changes.
-///               key: new ValueKey<int>(_count),
-///               textScaleFactor: 3.0,
+///       child: new Directionality(
+///         textDirection: TextDirection.rtl,
```

Comment:
Why is the text direction flipped here? Did you just want to reverse the column direction? If so, set an argument on the column.

Reply:
(otherwise e.g. any text here will be the wrong direction)

Reply:
Having it will make naive users cargo cult `Directionality`s into their code, which is worse because they don't get any error messages if they do that.